hmrc-mongo-play-27/src/main/scala/uk/gov/hmrc/mongo/play/json/CollectionFactory.scala

```diff
@@ -27,13 +29,17 @@ trait CollectionFactory {
   def collection[A: ClassTag](
     db: MongoDatabase,
     collectionName: String,
-    domainFormat: Format[A]
+    domainFormat: Format[A],
+    extraCodecs: Seq[Codec[_]]
```

Comment:
can we add a default value of Seq.empty here too?

Reply:
Absolutely 👍